framework/src/meshdivisions/CartesianGridDivision.C

```diff
@@ -77,16 +104,33 @@ CartesianGridDivision::divisionIndex(const Elem & elem) const
 unsigned int
 CartesianGridDivision::divisionIndex(const Point & pt) const
 {
+  unsigned int offset = 0;
+  // Determine the local grid bounds
+  Point bottom_left = _bottom_left, top_right = _top_right, p = pt;
```

Comment:
I would put this in a following `else` condition. I think it helps communicate that this line isn't used `if (_center_positions)`

Reply:
done